.circleci/config.yml

```diff
@@ -414,6 +414,11 @@ jobs:
       - run:
           name: Run mypy and report results
           command: nox -vs mypy-report
+      - run:
+          name: Run import sanity check
+          command: |
+            pip install .
+            python -c "import wandb"
```

Comment:
I think we want a new venv in the test. Maybe something like:

```
uv venv .venv-deps
source .venv-deps/bin/activate
uv pip install .
python -c 'import wandb'
```

Naming the `venv` helps avoid reusing an existing one. Then we can also create separate checks for the extras that we support.

Reply:
Yeah I can update that… do we want to use nox env instead or is it unnecessary?

Reply:
I think unnecessary